index.bs

```diff
@@ -139,7 +145,7 @@ urlPrefix: https://tc39.github.io/ecma262/; spec: ECMA-262
         text: sections 9.1; url: sec-ordinary-object-internal-methods-and-internal-slots
         text: 9.3.1; url: sec-built-in-function-objects-call-thisargument-argumentslist
         text: ECMA-262 section 9.3; url: sec-built-in-function-objects
-        text: function object; url: sec-built-in-function-objects
```

Comment:
The only place that links "function object" is [here](https://heycam.github.io/webidl/#ecmascript-binding), which already has another reference to that section:

> If an object is defined to be a function object, then it has characteristics as described in [ECMA-262 section 9.3].